app/src/main/res/values/strings.xml

```diff
@@ -113,6 +113,9 @@
     <string name="msg_report_not_sent">Report NOT sent</string>
     <string name="lbl_yes">Yes</string>
     <string name="msg_error_connecting_server">Error Connecting to Server</string>
+    <string name="msg_error_server_unavailable">Server unavailable</string>
+    <string name="msg_error_server_version">Unsupported server version. Server version must be %1$s or later.</string>
+    <string name="msg_error_signin">Error Signing In</string>
```

Comment:
If this is a toast I would make it a full sentence.

Reply:
@dkanada any suggestions on what to make this? This error message is displayed as a toast notification if automatic login with saved credentials fails.